src/libratbag.h

```diff
@@ -540,6 +540,18 @@ ratbag_device_has_capability(const struct ratbag_device *device,
 /**
  * @ingroup device
  *
+ * Write any profile changes to the device. This may take a couple of seconds
+ * to finish due to how slow most gaming mice are.
```

Comment:
"Write any changes to the device. Depending on the device, this may take up to a couple of seconds."